src/pages/workspaces/workspace/Workflows.js

```diff
@@ -301,6 +302,7 @@ export const Workflows = _.flow(
     super(props)
     this.state = {
       sortOrder: defaultSort.value,
+      filter: '',
       ...StateHistory.get()
```

Comment:
It might be good to also save the new filter in `StateHistory`, so it persists across page loads.